### PR TITLE
TSV-291 Update TAR throttle to use Credid instead of Userid

### PR DESCRIPTION
**Pull request by anshulbajpai**

### CODE REVIEW

File: app/services/TwoStepVerification.scala [hunk 1]
```diff
@@ -78,31 +78,35 @@ trait TwoStepVerification {
...
-            })
-          }
-        case _ => Future.successful(None)
+    def throttleLocation(rule: Biz2SVRule, locations: ThrottleLocations, credId: String) = twoStepVerificationThrottle.isRegistrationMandatory(rule.name, credId) match {
+      case true =>
+        if (isUplifted(locations.mandatory)) auditContext.setSentToMandatory2SVRegister(rule.name)
+        sendAuditEvent(rule, ruleContext, mandatory = true)
+        locations.mandatory
+      case _ =>
+        if (isUplifted(locations.optional)) auditContext.setSentToOptional2SVRegister(rule.name)
```
Comment (scottkwalker) [inline app/services/TwoStepVerification.scala]: is it right that nothing is set when isUplifted is false

Comment (anshulbajpai) [inline app/services/TwoStepVerification.scala]: We haven't changed any business logic. This question is out of scope of this PR. @paoloboni is best to answer. I wasn't here when uplifted location bit was implemented

Comment (scottkwalker) [inline app/services/TwoStepVerification.scala]: ok
